docs/configuration/relocation/README.md

```diff
@@ -210,6 +210,12 @@ To configure automatic dependency relocation, set `enableAutoRelocation = true`
 > Be mindful that some Gradle plugins will automatically add dependencies to your class path. You may need to remove these
 > dependencies if you do not intend to shadow them into your library.
 
+
+## Relocating Kotlin Standard Library and Reflect
+
+It is not recommended to relocate Kotlin Standard Library and Reflect because they are tightly coupled with Kotlin
```

Comment:
Let's rephrase to something like this: "It is not recommended to relocate Kotlin Standard Library if you are using Kotlin Metadata or Kotlin Reflect in the project ..."

I reformulated it this way because:
- for most simple cases, the stdlib relocation does not disrupt the application
- kotlin reflect can be relocated 
- In addition to Reflect, Kotlin Metadata may also be disrupted

Reply:
Done.